Dockerfile-jit

```diff
@@ -0,0 +1,34 @@
+FROM ghcr.io/graalvm/native-image-community:24.0.2
```

Comment:
@adamdickmeiss Using GraalVM also as the regular JVM? Interesting. It might have different perf characteristics compared to OpenJDK as it uses a different JIT under the hood, it would be worth to compare.

Also, I think we should switch to GraalVM Enterprise Edition which unlike the name suggests is free and offers better optimizations.

EE native image: `container-registry.oracle.com/graalvm/native-image:24`
Graal without native-image (smaller) to be used as a standard JDK: `container-registry.oracle.com/graalvm/jdk:24`